src/Schema.js

```diff
@@ -337,7 +337,7 @@ define(['src/options', 'src/storage', 'src/debug'], function (CSLEDIT_options, C
 
 			// add child results to the result list
 			$.each(node.childNodes, function (i, childNode) {
-				if (childNode.localName !== null) {
+				if (childNode.nodeName !== '#text') {
```

Comment:
I've got biased by surrounding lines. This is actually a test for text node. Could equally be `nodeType === 3`